lib/model/katex.dart

```diff
@@ -136,6 +133,123 @@ class _KatexParser {
   KatexNode _parseSpan(dom.Element element) {
     // TODO maybe check if the sequence of ancestors matter for spans.
 
+    if (element.className.startsWith('strut')) {
+      if (element.className == 'strut' && element.nodes.isEmpty) {
+        final styles = _parseSpanInlineStyles(element);
+        if (styles == null) throw KatexHtmlParseError();
+
+        final heightEm = styles.heightEm;
+        if (heightEm == null) throw KatexHtmlParseError();
+        final verticalAlignEm = styles.verticalAlignEm;
+
+        // Ensure only `height` and `vertical-align` inline styles are present.
+        if (styles.filter(heightEm: false, verticalAlignEm: false) !=
+            KatexSpanStyles()) {
```

Comment:
nit: binary operator goes after newline, not before (so it doesn't get visually lost off at the ragged right edge of the code):
```suggestion
        if (styles.filter(heightEm: false, verticalAlignEm: false)
            != KatexSpanStyles()) {
```